arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2508 @@
+        }
+      }
+    },
+    "ProcessCollection": {
+      "type": "object",
+      "description": "Collection of Process resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Process"
+          },
+          "description": "collection of Process Resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "PortCollection": {
+      "type": "object",
+      "description": "Collection of Port resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Port"
+          },
+          "description": "collection of Port resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "ClientGroupMembersCollection": {
+      "type": "object",
+      "description": "Collection of ClientGroupMember resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientGroupMember"
+          },
+          "description": "collection of ClientGroupMember resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "ConnectionCollection": {
+      "type": "object",
+      "description": "Collection of Connection resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Connection"
+          },
+          "description": "collection of Connection Resources"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "the URL to the next set of resources"
+        }
+      }
+    },
+    "MonitoringState": {
+      "description": "Used to specify if a resources is monitored or discovered.",
+      "type": "string",
+      "enum": [
+        "monitored",
+        "discovered"
+      ],
+      "x-ms-enum": {
+        "name": "MonitoringState",
+        "modelAsString": false
+      }
+    },
+    "VirtualizationState": {
+      "description": "Specifies if the machine is physical, virtual, hypervisor, or unknown.",
+      "type": "string",
+      "enum": [
+        "unknown",
+        "physical",
+        "virtual",
+        "hypervisor"
+      ],
+      "x-ms-enum": {
+        "name": "VirtualizationState",
+        "modelAsString": false
+      }
+    },
+    "Bitness": {
+      "description": "Specifies the bitness of a machine or process.",
+      "type": "string",
+      "enum": [
+        "32bit",
+        "64bit"
+      ],
+      "x-ms-enum": {
+        "name": "Bitness",
+        "modelAsString": false
+      },
+      "x-ms-external": true
```

Comment:
Why is the enum marked as `"x-ms-external": true`?
@selvasingh @johanste - Does this qualify to be a criteria in the checklist?